### ISSUE TITLE
Cannot invest focus point on lifestyle

### ISSUE THREAD

**Issue by VictorTadeuOliveira** [R-Vaccari/bannerlord-banner-kings#98]
**Describe the bug**
Title. Caravaneer lifestyle.
I have 100% progress, free focus point and way over the necessary skill on both ones needed. The button still is greyed out.

**Mod Version**
- Standard
- Numeric Version: 1.2.7, also on 1.2.7.1 (updated)

**Evidence**
- I tried disabling mods that I thought could conflict, like Diplomacy. Same issue. Could try disabling more if required.

**Crash Report**
No crash.

**Comment by VictorTadeuOliveira** [comment]
Changing lifestyle fix the issue. I can change to anothe lifestyle and back to the same one, keep the same progress and now spend the focus point. I'm keeping it here if someone else has this bug.

By the way I think lifestyle progress soooo slow... I already was over level 20, 300 trade skills and over 200 days when the firt progress was complete. Is it based on wisdom?

**Comment by UNOwenWasMe** [comment]
I can confirm this is the case, I noticed I had to change lifestyles before I could assign the focus point.

**Comment by SamJBarney** [comment]
I too am having this issue, but unfortunately changing lifestyles is not fixing it. I have Gladiator at 100%, and I switched to another one and back but am not able to assign the point.

**Comment by VictorTadeuOliveira** [comment]
> I too am having this issue, but unfortunately changing lifestyles is not fixing it. I have Gladiator at 100%, and I switched to another one and back but am not able to assign the point.

Do you have the required skills for that level? I think the first gladiator level is 80 in riding + athletics.

**Comment by tprossiter0** [comment]
> I too am having this issue, but unfortunately changing lifestyles is not fixing it. I have Gladiator at 100%, and I switched to another one and back but am not able to assign the point.

Yes I'm also having this issue. kinda bums me out. changing lifestyles hasn't worked since about a year into game.

**Comment by 21Arne42** [comment]
Sorry to necro but:
I've got the same issue in 1.2.9
The workaround in switching the lifestyle worked in my first test playthrough. Now that I have started the actual play, the workaorund doesn't work any more.
Tried moving it around with the load order but no joy. Question is: is this a bug from BK or a mod conflict? If it's a conflict, how do I find out which mod is causing it?
